From: Cristian Maureira-Fredes <Cristian.Maureira-Fredes@qt.io>
Date: Mon, 17 Oct 2022 09:57:22 +0200
Subject: make wheels compatible with Python 3.11

These leftover mentions to <3.11 made those wheel
impossible to install for Python 3.11

Pick-to: 6.4
Fixes: PYSIDE-2086
Change-Id: I2a0e3f87c265e3ddc97e1036ea0137a12e895794
Reviewed-by: Cristian Maureira-Fredes <cristian.maureira-fredes@qt.io>
(cherry picked from commit d169554f2e8e57382dfc66a62e06fa75706cb649)
---
 build_scripts/config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build_scripts/config.py b/build_scripts/config.py
index 63e52a2..7cbb8b6 100644
--- a/build_scripts/config.py
+++ b/build_scripts/config.py
@@ -137,7 +137,7 @@ class Config(object):
         setup_kwargs['zip_safe'] = False
         setup_kwargs['cmdclass'] = cmd_class_dict
         setup_kwargs['version'] = package_version
-        setup_kwargs['python_requires'] = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.11"
+        setup_kwargs['python_requires'] = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.12"
 
         if quiet:
             # Tells distutils / setuptools to be quiet, and only print warnings or errors.
